WordPressAuthenticator/Authenticator/WordPressAuthenticatorDisplayStrings.swift

```diff
@@ -12,19 +12,34 @@ public struct WordPressAuthenticatorDisplayStrings {
 
     public let siteLoginInstructions: String
 
+    /// Strings: primary call-to-action button titles.
+    ///
+    public let siteAddressPrimaryButton: String
+
     /// Designated initializer.
     ///
-    public init(emailLoginInstructions: String, jetpackLoginInstructions: String, siteLoginInstructions: String) {
+    public init(emailLoginInstructions: String,
+                jetpackLoginInstructions: String,
+                siteLoginInstructions: String,
+                siteAddressPrimaryButton: String) {
         self.emailLoginInstructions = emailLoginInstructions
         self.jetpackLoginInstructions = jetpackLoginInstructions
         self.siteLoginInstructions = siteLoginInstructions
+        self.siteAddressPrimaryButton = siteAddressPrimaryButton
```

Comment:
This seems to indicate we're going to have a string for every button on every view. That seems a bit excessive. Does the button text need to be customized? And to this level of granularity? It looks like host apps will typically use the same text for the same actions. 

If we do need to customize the button text, I'm wondering if maybe we can have more generic strings, like `continue`, `done`, `openMail` , i.e. customize the text for the action instead of the view.

What do you think?

Reply:
I'm overthinking it. More generic names for strings sounds better to me!